Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+node('master') {
+
+    Random random = new Random()
+    env.PROJECT = 'platform-dev'
+    tokens = "${env.WORKSPACE}".tokenize('/')
+    env.SITE_PATH = tokens[tokens.size()-1]
+    env.DB_NAME = "${env.PROJECT}".replaceAll('-','_').trim() + '_' + sh(returnStdout: true, script: 'date | md5sum | head -c 4').trim()
+    env.RELEASE_NAME = "${env.JOB_NAME}".replaceAll('%2F','-').replaceAll('/','-').trim()
+    env.HTTP_MOCK_PORT = random.nextInt(50000) + 10000
+
+    stage('Init') {
+        deleteDir()
+        checkout scm
+    }
+
+    stage('Build') {
+        withCredentials([
+            [$class: 'UsernamePasswordMultiBinding', credentialsId: 'mysql', usernameVariable: 'DB_USER', passwordVariable: 'DB_PASS'],
+            [$class: 'UsernamePasswordMultiBinding', credentialsId: 'flickr', usernameVariable: 'FLICKR_KEY', passwordVariable: 'FLICKR_SECRET']
+        ]) {
+            sh 'mysql -u $DB_USER --password=$DB_PASS -e "CREATE database $DB_NAME;"'
```

Comment:
I think this should be handled properly in a phing task and instead of calling shell commands, we should only use phing commands.